### PR TITLE
Model/client

### PR DESCRIPTION
**Pull request by kodowicz**
create reservation and its tickets

### CODE REVIEW

File: app/controllers/clients_controller.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+class ClientsController < ApplicationController
+  def show
+    client = Client
+      .where(email: [client_params[:email]])
+      .or(Client.where(id: params[:id]))
```
Comment (wmaciejak) [inline app/controllers/clients_controller.rb:5]: Nice usage of Active Records querying abilities 👌

File: app/controllers/reservations_controller.rb [hunk 2]
```diff
@@ -1,14 +1,12 @@
 # frozen_string_literal: true
```
Comment (wmaciejak) [inline app/controllers/reservations_controller.rb:1]: This controller looks like the perfect point to start refactorization to MonteWay - it's really bloated 😢

**Comment by wmaciejak** [comment]
From my perspective looks really good. 
1. Please rebase with mater - I see that you have conflicts.
2. Consider small test suite for some controllers. to be sure that everything works fine (for now it might be just checking if response status is 200.

File: app/controllers/reservations_controller.rb (hunk 2 above)
Comment (kodowicz) [reply app/controllers/reservations_controller.rb:1]: yeah, I can tell 😒 This is what I'm going to do next

**Pull request merged**